legal-api/src/legal_api/models/business.py

```diff
@@ -50,7 +50,8 @@ class LegalTypes(Enum):
         """Render an Enum of the Business Legal Types."""
 
         COOP = 'CP'
-        BCOMP = 'BC'
+        COMP = 'BC'
+        BCOMP = 'BEN'
```

Comment:
Do the UIs have to change anything?
See https://github.com/bcgov/business-filings-ui/blob/master/src/enums/legalTypes.ts

Reply:
There is ticket to update UIs.

https://github.com/bcgov/entity/issues/5048